src/patternfly/components/Form/form.scss

```diff
@@ -10,8 +10,7 @@
   --pf-c-form__label--FontWeight: var(--pf-global--FontWeight--normal);
   --pf-c-form__label--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-form__label--LineHeight: var(--pf-global--LineHeight--sm);
-  --pf-c-form__label--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-form__label--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__label--PaddingY: var(--pf-global--spacer--sm);
```

Comment:
Looks like you need to update https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/Form/form.scss#L66 and https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/Form/form.scss#L86